src/entries/popup/components/ImportWallet/ImportWalletSelection.tsx

```diff
@@ -69,17 +70,14 @@ const useDeriveAccountsFromSecrets = (secrets: string[]) => {
 };
 
 export const useImportWalletsFromSecrets = () => {
-  const [isImporting, setIsImporting] = useState(false);
-
-  const importSecrets = async ({
-    secrets,
-    accountsIgnored = [],
-  }: {
-    secrets: string[];
-    accountsIgnored?: Address[];
-  }) => {
-    setIsImporting(true);
-    return (async () => {
+  const { mutateAsync: importSecrets, isLoading: isImporting } = useMutation({
+    mutationFn: async ({
+      secrets,
```

Comment:
does this use cache to store results? if so maybe is not a good idea to use secrets for it